src/material-examples/datepicker-views-selection/datepicker-views-selection-example.html

```diff
@@ -0,0 +1,7 @@
+<mat-form-field>
+  <input matInput [matDatepicker]="dp" placeholder="Choose a year and a month" [formControl]="date">
+  <mat-datepicker-toggle matSuffix [for]="dp" (yearSelected)="chosenYearHandler($event)"
```

Comment:
These listeners should go on the datepicker, not the toggle

Reply:
ops!